angular.json

```diff
@@ -10,7 +10,7 @@
       "prefix": "lib",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-ng-packagr:build",
+          "builder": "@angular-devkit/build-angular:ng-packagr",
```

Comment:
why do we need to change the builder?

Reply:
Deprecated `@angular-devkit/build-ng-packagr` has been removed in angular 11

More details please see [build: delete @angular-devkit/build-ng-packagr(https://github.com/angular/angular-cli/commit/b1f08585f7a1df0baf66635d5187caca996ef902